apps/api/src/modules/faculty/faculty.module.ts

```diff
@@ -0,0 +1,13 @@
+import { FacultyService } from '@api/modules/faculty/faculty.service';
+import { Module } from '@nestjs/common';
+import { PrismaService } from '../persistance/prisma/prisma.service';
+import { FacultyController } from './faculty.controller';
+
+@Module({
+  imports: [],
+  providers: [FacultyService, PrismaService],
+  controllers: [FacultyController],
+  exports: [FacultyService],
+})
+// eslint-disable-next-line prettier/prettier
```

Comment:
prettier should not complain about this line.. is there any changes in the config files..?